src/main/java/io/fabric8/maven/docker/access/NetworkCreateConfig.java

```diff
@@ -0,0 +1,26 @@
+package io.fabric8.maven.docker.access;
+
+import org.json.JSONObject;
+
+public class NetworkCreateConfig {
+    final JSONObject createConfig = new JSONObject();
+    final String name;
```

Comment:
At the moment i have no usecase for using options for networks. We could extend the NetworkCreateConfig if we support a network goal.